index.html

```diff
@@ -0,0 +1,149 @@
+            <li><a href="#" class="header_icon"><img src="./image/Linkedin icon.png" alt="linkedin_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/m_icon.png" alt="m_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/github_icon.png" alt="github"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/V_icon.png" alt="V_icon"></a></li>
+        </ul>
+    </section>  
+    <section class="works">
+        <div class="works_card">
+            <img class="works_img" src="./image/works_1.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_2.png" alt="works_img">
+            <h3 class="works_title">Multi-Post<br>Stories</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_3.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. please implement it